src/test/java/api/requests/RequestsAPILocationTests.java

```diff
@@ -121,32 +114,21 @@ public void locationIncludedForMultipleRequests()
 
     List<JsonObject> fetchedRequestsResponse = requestsClient.getAll();
 
-    assertThat(fetchedRequestsResponse.size(), is(2));
-
-    JsonObject firstFetchedRequest = getRecordById(
-      fetchedRequestsResponse, firstRequest.getId()).get();
+    JsonObject temeraireRequest =
```

Comment:
Can you help me understand this change? 

How is it different to `getRecordById`? 

Why is the record representation removed from the response?